azure-core/azure/core/exceptions.py

```diff
@@ -73,48 +76,49 @@ def raise_with_traceback(self):
             self.__traceback__ = self.exc_traceback
             raise self
 
-
-class AzureLibraryError(AzureError):
```

Comment:
With the removal of these exceptions - do we no longer intend to define exceptions in azure-core for client-side errors (e.g. a user sending request data that fails to serialize)

Reply:
My suggestion is to use standard exceptions (e.g. ValueError) for these scenarios. But I'd love to get feedback on this proposal...

Reply:
Yeah I like that idea... off the top of my head I can't *think* of any scenarios that would fall out side of a TypeError/ValueError - but I'll take a quick scout through the code to see what errors are being raised where any whether any of them may require more thought.